src/build/rollup-importmap-plugin.ts

```diff
@@ -0,0 +1,51 @@
+import { Plugin } from "rollup";
+import fs from "node:fs/promises";
+import path from "node:path";
+import { Flags } from "../types";
+import { getGenerator, JspmError } from "../utils";
+
+export const RollupImportmapPlugin = async (flags: Flags): Promise<Plugin> => {
+  const generator = await getGenerator({ ...flags, freeze: true });
+  await generator.install();
+
+  return {
+    name: "rollup-importmap-plugin",
+    resolveId: async (id: string, importer: string) => {
+      if (importer?.startsWith("http") && id?.startsWith(".")) {
+        const proxyPath = new URL(id, importer).toString();
+        return { id: proxyPath, external: false };
+      }
```

Comment:
Can you share the reason for why this condition is needed?

Reply:
Not needed, it was to solve relative imports that comes from `https` content. But, this is solved by passing `parentURL` to the generator. Removed these lines now 👍